src/test/java/seedu/placebook/logic/commands/CommandResultTest.java

```diff
@@ -51,4 +51,31 @@ public void hashcode() {
         // different exit value -> returns different hashcode
         assertNotEquals(commandResult.hashCode(), new CommandResult("feedback", false, true, false).hashCode());
     }
+
+    @Test
+    public void isShowHelp() {
+        CommandResult commandResult = new CommandResult("feedback");
+        assertFalse(commandResult.isShowHelp());
+
+        CommandResult anotherCommandResult = new CommandResult("feedback", true, false, false);
+        assertFalse(commandResult.isShowHelp());
+    }
+
+    @Test
+    public void isExit() {
+        CommandResult commandResult = new CommandResult("feedback");
+        assertFalse(commandResult.isExit());
+
+        CommandResult anotherCommandResult = new CommandResult("feedback", false, true, false);
+        assertFalse(commandResult.isExit());
+    }
+
+    @Test
+    public void isUndo() {
+        CommandResult commandResult = new CommandResult("feedback");
+        assertFalse(commandResult.isUndo());
+
+        CommandResult anotherCommandResult = new CommandResult("feedback", false, false, true);
+        assertFalse(commandResult.isUndo());
```

Comment:
Do we need to ‘assertTrue(anotherCommandResult.isUndo())’ here?

Reply:
Looks like there are duplicated 'assertFalse(commandResult.isUndo())' here. It is checking whether commandResult is affected by something?